app/src/lib/shell.ts

```diff
@@ -0,0 +1,85 @@
+/* tslint:disable:no-sync-functions */
+
+import { spawnSync } from 'child_process'
+import * as os from 'os'
+
+type IndexLookup = {
```

Comment:
We probably don't need this given `process.env` is `any` and we don't need to enumerate this in our code, but this felt like a bit of nice correctness to have defined on the method here.